From 467ef22cceb562ea6342b0224e8272e2f533bd62 Mon Sep 17 00:00:00 2001
From: Fabian Boehm <FHomborg@gmail.com>
Date: Mon, 23 Dec 2024 17:01:04 +0100
Subject: [PATCH 1/3] Remove SIGUNUSED

It is, as the name implies, unused - it became SIGSYS, which we
already check.

Since it is entirely undefined on some architectures it causes a build
failure there, see discussion in #10633
---
 src/signal.rs | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/signal.rs b/src/signal.rs
index 6aa995f98..9b51e6dfe 100644
--- a/src/signal.rs
+++ b/src/signal.rs
@@ -407,10 +407,6 @@ const SIGNAL_TABLE : &[LookupEntry] = &[
     #[cfg(target_os = "linux")]
     LookupEntry::new(libc::SIGIOT,   L!("SIGIOT"), L!("Abort (Alias for SIGABRT)")),
 
-    #[cfg(target_os = "linux")]
-    #[allow(deprecated)]
-    LookupEntry::new(libc::SIGUNUSED, L!("SIGUNUSED"), L!("Unused signal")),
-
     #[cfg(target_os = "linux")]
     LookupEntry::new(libc::SIGPWR,    L!("SIGPWR"), L!("Power failure")),
 
-- 
2.47.1

